addons/hr_timesheet/report/hr_timesheet_report_view.xml

```diff
@@ -81,11 +81,28 @@
         <record id="hr_timesheet_report_search" model="ir.ui.view">
             <field name="name">timesheets.analysis.report.search</field>
             <field name="model">timesheets.analysis.report</field>
-            <field name="inherit_id" ref="hr_timesheet.hr_timesheet_line_search_base"/>
```

Comment:
Shouldn't this still be inherited from `hr_timesheet.hr_timesheet_line_search`?

Reply:
Nope, `hr_timesheet.hr_timesheet_line_search_base` is being removed with the `[REV]` commit :wink: